addons/http_routing/views/http_routing_template.xml

```diff
@@ -100,22 +100,43 @@
         </t>
     </template>
 
-    <template id="404">
+    <template id="404" name="Page not found">
+        <!-- This template needs this odd `oe_structure_xyz` structure to be
+             fully editable (drag & dropable) by the editor. -->
         <t t-call="web.frontend_layout">
             <div id="wrap">
                 <t t-raw="0"/>
-                <div class="oe_structure oe_empty">
+                <div class="oe_structure" id="oe_structure_404_1"/>
+                <div class="oe_structure" id="oe_structure_404_2"/>
+                <div class="oe_structure"/>
```

Comment:
add the big big padding here... 
and not on the 404_2

why not an id oe_structure_x ? on this third div?

Reply:
- The first `oe_structure` is needed since we want its content to be fully editable.
- The second `oe_structure` is needed  to be inherited by other modules (not used in http_routing), not that it won't be able to receive drag & drop since it will be inherited, doing that will be disabled. That's the only limitation of the edition on the 404 but is fine.
- The last `oe_structure` is just to be able to add other elements with drag & drop and not just be limited to the first one, the ID will be handled/added automatically during the drag & drop. 

About the `padding`, where do you want it? The `oe_structure_404_2` is replaced by xpath.

Reply:
Added the `id` and created an empty xpath template on it to add `pt104` so it is fully customizable (previously it was a `pb104` on `oe_structure_404_2`).